### PR TITLE
More parameters

### PR DESCRIPTION
**Pull request by edap**
I've removed the previous pull request because it was pinting to my master. On this branch there are only the commits relatives to the addition of new parameters.

### CODE REVIEW

File: deepdreams.py [hunk 1]
```diff
@@ -60,7 +60,7 @@ def make_step(net, step_size=1.5, end='inception_4c/output', jitter=32, clip=Tru
 		src.data[:] = np.clip(src.data, -bias, 255-bias)
 
 def deepdream(net, base_img, end, iter_n=10, octave_n=4, octave_scale=1.4, clip=True, **step_params):
-	# prepare base images for all octaves
```
Comment (mjibson) [inline deepdreams.py]: Revert this please

File: deepdreams.py [hunk 2]
```diff
@@ -99,4 +99,19 @@ def deepdream(net, base_img, end, iter_n=10, octave_n=4, octave_scale=1.4, clip=
 if len(sys.argv) >= 2:
 	end = sys.argv[1]
 
-_=deepdream(net, img, end)
+iter_n = 10
+if len(sys.argv) >= 3:
+	iter_n = sys.argv[2]
+        iter_n = int(iter_n)
```
Comment (mjibson) [inline deepdreams.py]: I use tabs instead of spaces. This is broken python because the whitespace is different.

**Comment by mjibson** [comment]
I've merged your changes with you as author.

File: deepdreams.py (hunk 2 above)
Comment (edap) [inline deepdreams.py]: oh sorry, I'm totally new to python

**Pull request closed without merging**